app/views/admin/categories/_status.html.erb

```diff
@@ -1,7 +1,7 @@
 <%= content_tag :div, class: 'loader-btn' do %>
   <% if locals[:category].enabled %>
```

Comment:
try this..
text, enabled = locals[:category].enabled ? ['disable', false] : ['enable', true]